packages/@dcl/inspector/src/components/tree/Tree.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import { useDrag, useDrop } from 'react-dnd'
+
+import './Tree.css'
+
+import { Tree, TreeType } from '../../utils/tree'
+import { Input } from './input'
+import { Controls } from './controls'
+import { Icon } from './icon'
+
+interface Props {
+  value: Tree
+}
+
+interface TreeProps {
+  tree: Tree
+  update: () => void
+}
+
+const getEditModeStyles = (active: boolean) => ({ display: active ? 'none' : 'block' })
+const getExpandStyles = (active: boolean) => ({ height: active ? 'auto' : '0', overflow: 'hidden', display: 'block' })
+
+const MemoTree = React.memo(TreeComponent)
+
+function TreeComponent({ tree, update }: TreeProps) {
+  const { value, childs, parent, expanded = false, type } = tree
```

Comment:
The `expanded` value should be stored in the component's state, since it's not part of the data itself from the tree